### PR TITLE
new Plugin: FriendsSince ~ shows friend date in profiles

### PR DESCRIPTION
**Pull request by EdVraz**
Shows the Date you and someone else became friends in the user popout.

Implements https://github.com/Vencord/plugin-requests/issues/157

image

### CODE REVIEW

File: src/plugins/friendsSince/index.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        }
+    ],
+
+    friendsSince (data: UserPopoutData, locale: string) {
+        const { userId, headingClassName, textClassName } = data;
+        const friendsSince = RelationshipStore.getSince(userId);
+        if (!friendsSince) return;
+
+        return <React.Fragment>
+            <div style={{ height: ".65em" }}/>
```
Comment (Vendicated) [inline src/plugins/friendsSince/index.tsx:45]: use a margin on the Heading element below instead

File: src/plugins/friendsSince/index.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <Heading variant="eyebrow" className={headingClassName}>
+                {"Friends Since"}
```
Comment (Vendicated) [inline src/plugins/friendsSince/index.tsx:47]:
```suggestion
                Friends Since
```

File: src/plugins/friendsSince/index.tsx (hunk 1 above)
Comment (Vendicated) [reply src/plugins/friendsSince/index.tsx:45]: or rather dont do any of this and instead use discord's classes and it'll automatically do it correctly

Comment (EdVraz) [reply src/plugins/friendsSince/index.tsx]: done

**Pull request merged**